src/plugins/blocksmith/components/imgur-post-block-preview/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Message from '@gdbots/pbj/Message';
+import ImgurPostBlockEmbed from './embed';
```

Comment:
@jalopez13 name the file `ImgurPostBlockBlockquote.jsx` or just `Blockquote.jsx`

Reply:
@efjacobson 

Renamed to Blockquote.jsx